packages/ui/src/components/theme/styleOverrides/drawerStyle.ts

```diff
@@ -0,0 +1,28 @@
+import { DrawerClasses, Theme } from '@mui/material';
+import { OverridesStyleRules } from '@mui/material/styles/overrides';
+
+import { blurEffects, palette } from '../themeOptions';
+
+export const DRAWER_WIDTH = 16.25;
+
+export const drawerStyle:
+    | Partial<OverridesStyleRules<keyof DrawerClasses, 'MuiDrawer', Omit<Theme, 'components'>>>
+    | undefined = {
+    docked: {
+        width: `${DRAWER_WIDTH}rem`,
```

Comment:
you can store the width in rem and not have to do string interpolation, `const DRAWER_WIDTH_REM`